src/angularjs/src/app/places/list/places-list-map-directive.js

```diff
@@ -0,0 +1,80 @@
+(function() {
+
+    /* @ngInject */
+    function PlacesListMapController(MapConfig, Neighborhood) {
```

Comment:
Bummer. In looking at this and `neighborhood-map.directive.js` it seems like they're just different enough that using the same map directive that pulls the neighborhoods geojson would take a bit more work.